alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+RUN apk --no-cache add curl python
```

Comment:
Please consider adding git to the list of packages otherwise the gcloud source repo commands do not work.